osu.Game.Rulesets.Osu/Skinning/Legacy/LegacyReverseArrow.cs

```diff
@@ -12,33 +13,55 @@ namespace osu.Game.Rulesets.Osu.Skinning.Legacy
 {
     public class LegacyReverseArrow : CompositeDrawable
     {
-        private ISkin skin { get; set; }
+        private ISkin skin { get; }
 
         [Resolved(canBeNull: true)]
         private DrawableHitObject drawableHitObject { get; set; }
 
+        private Drawable proxy;
+
         public LegacyReverseArrow(ISkin skin)
         {
             this.skin = skin;
         }
 
-        [BackgroundDependencyLoader]
+        [BackgroundDependencyLoader(true)]
```

Comment:
```suggestion
        [BackgroundDependencyLoader]
```

Reply:
oh yep, that's a vestige that i forgot to trim, will remove